default.htm

```diff
@@ -127,129 +136,395 @@ <h3>Keep Talking and Nobody Explodes</h3>
+				document.getElementById("basicwires_red0").checked = true;
+				document.getElementById("basicwires_blue0").checked = true;
+				document.getElementById("basicwires_yellow0").checked = true;
+				document.getElementById("basicwires_black0").checked = true;
+				document.getElementById("basicwires_white0").checked = true;
+				document.getElementById("button_serial1").checked = true;
+				document.getElementById("basicwires_lastblue").checked = true;
 
-					Array.from(document.querySelectorAll('.btn-group-vertical .btn')).forEach(element => {
-						element.addEventListener('click', buttonDeselect);
-					});
-				}
+
+			}
 
 
-				const lookups = [
-					'about', 'after', 'again', 'below', 'could',
-					'every', 'first', 'found', 'great', 'house',
-					'large', 'learn', 'never', 'other', 'place',
-					'plant', 'point', 'right', 'small', 'sound',
-					'spell', 'still', 'study', 'their', 'there',
-					'these', 'thing', 'think', 'three', 'water',
-					'where', 'which', 'world', 'would', 'write'
+			const lookups = [
+				'about', 'after', 'again', 'below', 'could',
+				'every', 'first', 'found', 'great', 'house',
+				'large', 'learn', 'never', 'other', 'place',
+				'plant', 'point', 'right', 'small', 'sound',
+				'spell', 'still', 'study', 'their', 'there',
+				'these', 'thing', 'think', 'three', 'water',
+				'where', 'which', 'world', 'would', 'write'
+			];
+
+			function passwd_solve(e) { //Rename to sPassords or something?
+				var inputs = [
+					document.getElementById("passwd_letters1").value.toLowerCase().split(''),
+					document.getElementById("passwd_letters2").value.toLowerCase().split(''),
+					document.getElementById("passwd_letters3").value.toLowerCase().split(''),
+					document.getElementById("passwd_letters4").value.toLowerCase().split(''),
+					document.getElementById("passwd_letters5").value.toLowerCase().split('')
 				];
+				let matches = lookups.filter(lookupWord => {
+					let wordLetters = lookupWord.split('');
 
-				function solve(e) {
-					var inputs = [
-						document.getElementById("letters1").value.toLowerCase().split(''),
-						document.getElementById("letters2").value.toLowerCase().split(''),
-						document.getElementById("letters3").value.toLowerCase().split(''),
-						document.getElementById("letters4").value.toLowerCase().split(''),
-						document.getElementById("letters5").value.toLowerCase().split('')
-					];
-					let matches = lookups.filter(lookupWord => {
-						let wordLetters = lookupWord.split('');
-
-						let hasMatch = true;
-						for (var i = 0; i < 5; i++) {
-							hasMatch &&= inputs[i].length === 0 || inputs[i].includes(wordLetters[i]);
-						}
-						return hasMatch;
+					let hasMatch = true;
+					for (var i = 0; i < 5; i++) { //I've been drinking but could you replace '5' condition with inputs[i].length and remove the === 0 below? SM
+						hasMatch &&= inputs[i].length === 0 || inputs[i].includes(wordLetters[i]);
```

Comment:
You're getting muddled up between inputs[i].length and inputs.length.
This like could be changed to `for (var i = 0; i < inputs.length; i++) {` but the one below is getting the number of chars in inputs[i] to see if any have been put in.

Reply:
Yeah, I read this today and I was wrong.